docs/site/BelongsTo-relation.md

```diff
@@ -442,17 +444,144 @@ export class OrderRepository extends DefaultCrudRepository {
   ];
   ```
 
-  Here is a diagram to make this more intuitive:
+{% include note.html content="The query syntax is a slightly different from LB3. We are also thinking about simplifying the query syntax. Check our GitHub issue for more information: [Simpler Syntax for Inclusion](https://github.com/strongloop/loopback-next/issues/3205)" %}
+
+Here is a diagram to make this more intuitive:
 
-  ![Graph](./imgs/belongsTo-relation-graph.png)
+![Graph](./imgs/belongsTo-relation-graph.png)
 
 - You can delete a relation from `inclusionResolvers` to disable the inclusion
   for a certain relation. e.g
   `orderRepository.inclusionResolvers.delete('customer')`
 
-{% include note.html content="
-Inclusion with custom scope:
-Besides specifying the relation name to include, it's also possible to specify additional scope constraints.
-However, this feature is not supported yet. Check our GitHub issue for more information:
-[Include related models with a custom scope](https://github.com/strongloop/loopback-next/issues/3453).
+### More use cases - query multiple relations:
```

Comment:
Since we're duplicating the contents for the 3 relations pages, should we put it at the top level to avoid duplication?

Reply:
It doesn't make sense to me if we put it at `Relation.md`. How about I link this part from `belongsTo` and `hasOne` to `hasMany`?